docs/developer-docs/latest/setup-deployment-guides/configurations.md

```diff
@@ -1575,3 +1575,108 @@ module.exports = () => ({
   },
 });
 ```
+
+### Role-Based Access Control <BronzeBadge withLinkIcon link="https://strapi.io/pricing" />
```

Comment:
I'm wondering if we should put the Bronze, Silver, and Gold badges here as while it is assumed that if you have a higher tier license it would also work (unlikely we would ever release a feature that is only for bronze and not silver or gold). It may not be clear to the user. :thinking:

Reply:
That's a great suggestion! Should we:
* either include the 3 badges 
* or create a EE badge?

I think the 3 badges is better, what's your take on this @meganelacheny ?

Reply:
In the User Guide I only used the Bronze badge and explained in the introduction how badges work. I hesitated because I'm not sure that's the best solution. I don't think we should use "EE" because I'm not sure it's clear enough and it's weird to have an "EE" badge and a "Gold" badge 🤔
Perhaps @derrickmehaffy is right and we should just put all badges, it's clear for everyone and it avoids adding information in the introduction. Of course no matter what we choose we should do the same thing in both the developer docs and the user guide.